projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -37,10 +37,10 @@ import { Arc, arc, DefaultArcObject } from 'd3-shape';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class GaugeComponent implements OnChanges {
-  private static readonly GAUGE_RING_WIDTH: number = 20;
+  private static readonly GAUGE_RING_WIDTH: number = 15;
   private static readonly GAUGE_ARC_CORNER_RADIUS: number = 10;
-  private static readonly GAUGE_AXIS_PADDING: number = 30;
   private static readonly GAUGE_MIN_RADIUS_TO_SHOW_PATH: number = 80;
+  private static readonly EXTRA_ARC_ANGLE: number = Math.PI / 12;
```

Comment:
Is the extra arc angle added to `start` and `end` so that the label appears more inside? Instead of this, could we have done this by updating the position of `text` on line 30?